aten/src/ATen/ScalarOps.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "ATen/core/Scalar.h"
+#include "ATen/Tensor.h"
+
+namespace at {
+
+// FIXME: this should be (and was) Scalar::toTensor, but there is currently no way
```

Comment:
Based on the discussion we had in the meeting today, it seems likely that eventually, there will be *some* way in ATen/core to allocate a vanilla CPU tensor. So when that happens, we can put this back in Core.